src/main/java/duke/command/ToDoCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyToDoException;
+import duke.tasks.TaskList;
+import duke.tasks.ToDo;
+import duke.ui.Ui;
+
+public class ToDoCommand extends UserCommand {
+    public ToDoCommand(String userInput) {
+        super(userInput);
+    }
+
+    /**
+     * @param taskList task list containing all the tasks.
+     * @param ui       ui that prints output.
```

Comment:
Should the abundant spaces between the parameter and its description be removed?